### ISSUE TITLE
Explicitly disable use of grammarly across CoCalc (was: Enable use of grammarly chrome extension)

### ISSUE THREAD

**Issue by dpotter** [sagemathinc/cocalc#4207]
I regularly use the Grammarly Chrome browser extension to check my grammar and spelling.  Would it be possible to adjust the CoCalc UI text boxes to allow it to operate?

[text box requirements](https://support.grammarly.com/hc/en-us/articles/115000090392-I-do-not-see-the-G-icon-on-a-certain-web-page-or-in-a-certain-text-field-but-see-it-on-other-web-pages-) :  

Below is a screenshot of the extension working on this text when I typed it.
<img width="1203" alt="Screen Shot 2019-11-01 at 8 50 31 PM" src="https://user-images.githubusercontent.com/1454072/68063754-dd2ffd00-fce9-11e9-804a-3ad15a08cf48.png">

**Comment by haraldschilly** [comment]
which parts of the CoCalc UI do you mean? e.g. Grammarly works in chat.

**Comment by haraldschilly** [comment]
I briefly looked into this. For CodeMirror inputs, the option `inputStyle: "contenteditable"` might cause it to be active, but it's not really working? I also have no idea about what implications this setting has.

**Comment by williamstein** [comment]
1. I think he wants this for Jupyter notebooks.

2. [This issue](https://github.com/hackmdio/codimd/issues/389) seems to think it is impossible to use Codemirror + grammarly.

3. We could implement spell check ourselves for codemirror.  We have "underline misspelled words" already (we don't use it for markdown cells in jupyter but could), and could add something to show corrections...

**Comment by williamstein** [comment]
Actually, what we should do is somehow change all of cocalc disable use of Grammarly.   That extension mysteriously and randomly crashes things all over cocalc, but there is a way using CSS (I think) to disable it site-wise.  There's a lot in these issues about how to do this:   https://github.com/ianstormtaylor/slate/issues?q=is%3Aissue+grammarly+is%3Aclosed

**Comment by williamstein** [comment]
- I tried all of the data-gramm style attributes, and none of them disable Grammarly anymore.
- I tried using this CSS, and it does seem to work to make Grammarly not appear:
```
grammarly-extension {
  display: none;
}
```
- However, here is the thing.  I wasn't able to reproduce any cases where Grammarly crashed cocalc or where making replacements didn't work, at least in our fork of slate-react.  For example, even this recent issue https://github.com/ianstormtaylor/slate/pull/4809 doesn't seem to happen for me when I try it in cocalc.

- So I'm going to close this issue until there is a reproducible case where Grammarly actually crashes cocalc.  I've heard reports that there are, but I've never seen one.
